multi/imxrt-multi/imxrt-multi.c

```diff
@@ -296,6 +307,14 @@ static int createDevFiles(void)
 		return -1;
 #endif
 
+#if CM4
+	for (i = 0; i < 4; ++i) {
+		sprintf(name, "cpuM4%d", i);
+		if (mkFile(&dir, id_cm4_0 + i - 1, name, multi_port) < 0)
+			return -1;
+	}
```

Comment:
```suggestion
	for (i = 0; i < 4; ++i) {
		sprintf(name, "cpuM4%d", i);
		if (mkFile(&dir, id_cm4_0 + i - 1, name, multi_port) < 0) {
			return -1;
		}
	}
```
Since `i < 10` this can be done easily without `sprintf()`.

Reply:
Wanted to stick to the style of the rest of the file, but fixed (along with other missing brackets)